aws/terraform/modules/metaflow/modules/computation/variables.tf

```diff
@@ -130,8 +130,7 @@ variable "batch_gpu_instance_types" {
   type        = list(string)
   description = "EC2 instance types allowed for GPU Batch jobs. Types can be explicitly or implicitly requested by data scientists."
   default = [
-    "p3.2xlarge",
-    "g4dn.8xlarge"
+    "p3.2xlarge"
```

Comment:
This was something you changed in our stack I believe that I am porting over. I can't remember the exact issue but it was causing jobs to get stuck in runnable under some conditions

Reply:
Ah yup, you're totally right! This was because we needed more RAM from the G4DN machine, but P3 worked out great when we relaxed our RAM requirements. Great memory haha